.github/workflows/main.yml

```diff
@@ -1,30 +1,112 @@
-# This is a basic workflow to help you get started with Actions
-
 name: CI
 
-# Controls when the action will run. 
 on:
-  # Triggers the workflow on push or pull request events but only for the master branch
   push:
-    branches: [ master ]
+    branches:
+      - master
   pull_request:
-    branches: [ master ]
+    branches:
+      - master
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 0 * * 2'
```

Comment:
I don't care about the special char, 
I care about the intention of the schedule: 
Please add: "triggers weekly on every Tuesday at night"

Reply:
Good point! Fixed!